esmvaltool/cmorizers/obs/cmorize_obs_woa.py

```diff
@@ -4,21 +4,28 @@
    Tier 2: other freely-available dataset.
 
 Source
-   https://data.nodc.noaa.gov/woa/WOA13/DATAv2/
+   WOA13: https://data.nodc.noaa.gov/woa/WOA13/DATAv2/
+   WOA18: https://www.nodc.noaa.gov/OC5/woa18/woa18data.html
 
 Last access
-   20190131
+   WOA13: 20190131
+   WOA18: 20200911
 
 Download and processing instructions
    Download the following files:
+     WOA13:
      temperature/netcdf/decav81B0/1.00/woa13_decav81B0_t00_01.nc
      salinity/netcdf/decav81B0/1.00/woa13_decav81B0_s00_01.nc
      oxygen/netcdf/all/1.00/woa13_all_o00_01.nc
      nitrate/netcdf/all/1.00/woa13_all_n00_01.nc
      phosphate/netcdf/all/1.00/woa13_all_p00_01.nc
      silicate/netcdf/all/1.00/woa13_all_i00_01.nc
```

Comment:
The instructions to download the WOA13 data remain but it is not possible to cmorize this data anymore. For the variables `no3`, `o2`, `po4` and `si` there will not be a possibility apply the cnorizer anymore. Is that a problem?